Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.14
+
+ENV GOFLAGS=-mod=readonly
+
+WORKDIR /go/src/github.com/terraform-providers/terraform-provider-grafana
+
+COPY . .
```

Comment:
Seems like we don't need this since docker-compose mounts the codebase in this path from the host machine.

Reply:
done